scripts.js

```diff
@@ -1,24 +1,64 @@
-/* For picking letters from alphabet keypad. Returns the
-   letter chosen or "already picked"
-*/
-function pickLetter(letter) {
-  "use strict";
-  var result;
-
-  if (letter.className === "letter-button") {
-    result = letter.innerHTML;
-    letter.className = "letter-disabled";
-  } else {
-    result = "already picked";
-  }
-
-  console.log(result);    // for testing, remove later
-  return result;
+/*
+ * Pick from alphabet keypad. Returns the letter chosen.
+ */
+$("#alphabet-keypad").on("click", ".letter-button", pickLetter);
+
+function pickLetter() {
+  var letterChosen = $(this);
+  
+  letterChosen
+    .removeClass("letter-button")
+    .addClass("letter-disabled");
+
+  // Next two lines to be refactored into interface for
+  //   checking if the letter is in the word.
+  
+  letterChosen = letterChosen.html();
+  
+  verify(letterChosen);
 }
 
-// adding dictionary and word filter //
+// placeholder for verifying the letter
+function verify(letter) {
+  console.log(letter);
+}
 
 
+/*
+ * Hangman Object with methods loseLife() and reset()
+ */
+var hangmanObject = function () {
+  var livesLost = 0,
+      maxLife = 7;
+  
+  return {
+    loseLife : function () {
```

Comment:
The idea now is we can combine this with @piecedigital changes. Whenever he puts a letter in the graveyard, he'll also call hangmanGraphic.addBodyPart() so the graphic corresponds to the gameplay.

Also whoever is working on the part that resets the game can call hangmanGraphic.reset() to reset the graphic as well.